msal/build.gradle

```diff
@@ -288,7 +288,8 @@ dependencies {
     androidTestImplementation "org.mockito:mockito-android:$rootProject.ext.mockitoAndroidVersion"
     // 'local' flavor dependencies
     localApi(project(":common")) {
-        transitive = false
+        transitive = true
```

Comment:
As I understand it, we have 2 options here right?
- Consume the dependency transitively or....
- Add the dependency directly

Is there any particular reason to favor a transitive consumption model?

Reply:
Yes, we have both options. I think for now since this is just source either should be fine but I think once this gets published my idea behind transitive was that msal shouldn't need to think about which version of `common4j` to consume. MSAL just consumes a specific version of `common` and then that specific version of common will know which version of `common4j` to use.

Do you see any risks with that?

Reply:
Do you know what is considered as best practice in regards to this? Seems in our current MSAL we exclude transitives when consuming common, however, when we consume MSAL in MSAL Test App then we don't take a direct dependency on common but rather let transitive do its thing.

Reply:
I would prefer this to be hidden from consumers.

Reply:
@rpdome So to translate that - you are in the `transitive` boat as well?